ci/vmcheck-provision.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+# Install build dependencies, run unit tests and installed tests.
+
+set -xeuo pipefail
+
+dn=$(dirname $0)
+. ${dn}/libbuild.sh
+pkg_install openssh-clients ansible
```

Comment:
FWIW, PAPR will automatically install `openssh-clients` in the container if provisioning a `cluster`. No harm in having it here though!